recipes/colormap/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: colormap
+  version: "1.0.0"
+
+source:
+  fn: colormap-1.0.0.tar.gz
+  url: https://pypi.python.org/packages/dd/de/2ab8312cdf4d4c2bc59b6fef4c51bf616dd431251d858079678c84c6359e/colormap-1.0.0.tar.gz
+  sha256: eb73ca956770ae4e92a071d53daeb3c20d02c8b167797a2784f739db09749edd
```

Comment:
Please update the section above this comment to match the [`example`]( https://github.com/conda-forge/staged-recipes/blob/e6bcccf6a72c3ca4411817f89e729c951a048462/recipes/example/meta.yaml#L1-L19 ).